lib/patches/rake-compiler/rake/baseextensiontask.rb

```diff
@@ -0,0 +1,17 @@
+Truffle::Patching.require_original __FILE__
+
+module Truffle::Patching::BaseExtensionTaskOverrideBinaryExt
```

Comment:
Could you add a Rake prefix to these modules?
They don't seem likely to clash so far but it's safer.

Reply:
`Rake` prefix added.